src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -102,10 +103,16 @@ public function putOffExpiration(Key $key, $ttl)
         $successCount = 0;
         $failureCount = 0;
         $storesCount = count($this->stores);
+        $expireAt = microtime(true) + $ttl;
 
         foreach ($this->stores as $store) {
             try {
-                $store->putOffExpiration($key, $ttl);
+                if (0.0 >= $adjustedTtl = $expireAt - microtime(true)) {
+                    $this->logger->warning('TTL expires during the put off the expiration of the "{resource}" lock.', array('resource' => $key, 'store' => $store, 'ttl' => $ttl));
```

Comment:
We are trying to put off the expiration of the lock (to extend it life duration) but the method take more time to perform that task than the asked TTL.  

For instance, when calling `putOffExpiration($key, 31)` takes 31 seconds.